src/Controller/Component/SecurityComponent.php

```diff
@@ -175,7 +176,7 @@ public function blackHole(Controller $controller, $error = '', SecurityException
      * @throws \Cake\Http\Exception\BadRequestException
      * @return void
      */
-    protected function _throwException($exception = null)
+    protected function _throwException(?\Cake\Controller\Exception\SecurityException $exception = null): void
     {
```

Comment:
Are we able to use the imported class name that isn't fully qualified?

Reply:
We should always have the FQN in the use statement IMO.

Reply:
Definitely, I overlooked this one.